packages/suite/src/views/suite/SwitchDevice/DeviceItem/EjectButton.tsx

```diff
@@ -31,7 +31,7 @@ export const EjectButton = ({ setContentType, dataTest }: EjectButtonProps) => {
 
     return (
         <EjectContainer $elevation={elevation}>
-            <Tooltip hasArrow cursor="pointer" content={<Translation id="TR_EJECT_HEADING" />}>
```

Comment:
Not sure if it shall be removed from all places where it is used explicitly. 

If its a place where you saw the render is ugly, its fine, but you just removed it everywhere I think somewhere it was intentionally used. 

Definitely you should check the visual of those places if it really was broken

Reply:
True. I just checked it briefly like here where it was definitely good to remove it.
<img width="303" alt="Screenshot 2024-06-19 at 11 46 06 AM" src="https://github.com/trezor/trezor-suite/assets/66002635/023363a9-cfe0-48b7-bdee-3ba9acf7a0c9">

Since it renders every time to the same place I think I can't be correct anywhere.

Reply:
Doesn't matter here, it's going to be removed soon